src/ocaml-output/Makefile

```diff
@@ -272,7 +272,9 @@ package: all
 	cp ../../version.txt $(PREFIX)/
 	cp ../../_tags $(PREFIX)/
 	cp ../../.common.mk $(PREFIX)/
+	cp ../../fsharp.extraction.targets $(PREFIX)/
 	@# Then the rest of the static files.
+    $(call fexport,.nuget)
```

Comment:
Just this and a line above are part of my change around makefiles. I need few files to be able to create a nuget and for F# extraction to work with a windows binary package. 

@tahina-pro I wasn't able to build a zip with the package locally. I've tried running `make -j6 -C ../.. package` from src/ocaml-output, but zip wasn't produced. Am I missing something here? 

For tests I've just copied the extra three files manually into downloaded binary package.

Reply:
With `make package` from the FStar root directory, a .zip should be created in `src/ocaml-output`. It is named with the version number, the OS and the platform. Alternatively, you can do `make package_unknown_platform` to produce a .zip with a simpler name, `src/ocaml-output/fstar.zip`.

Reply:
Thanks, it turned out I also had some issues with Madoko, but it works ok now!